frontend/app/src/store/utils.ts

```diff
@@ -163,3 +163,16 @@ export function useStore(): Store<RotkehlchenState> {
   assert(store);
   return store;
 }
+
+const KEY_ANIMATIONS_ENABLED = 'rotki.animations_enabled' as const;
+export function isAnimationsEnabled(): boolean {
+  return !localStorage.getItem(KEY_ANIMATIONS_ENABLED) ?? true;
+}
+
+export function setAnimationsEnabled(enabled: boolean): void {
+  if (!enabled) {
+    localStorage.setItem(KEY_ANIMATIONS_ENABLED, `${enabled}`);
+  } else {
+    localStorage.removeItem(KEY_ANIMATIONS_ENABLED);
+  }
```

Comment:
In the future, we could make our own reactive property and use that instead, but at the moment I see that this will not be possible since `FrontendSettings.vue` is still using the class component.